css/index.css

```diff
@@ -2,57 +2,56 @@
 :root{
 	--grey: #565656;
 	--magenta: #cc1e44;
-}
-@font-face{
-	font-family: news;
-	src: url("../assets/fonts/chomsky.otf");
-}
-@font-face{
-	font-family: letters;
-	src: url("../assets/fonts/italiana.ttf");
-}
-@font-face{
-	font-family: news1;
-	src: url("../assets/fonts/ChakraPetch-Bold.ttf");
+	--lightgrey:rgb(188, 188, 188);
 }
 @font-face {
-	font-family: AlfaSalb;
-	src: url("../assets/fonts/AlfaSlabOne-Regular.ttf");
+    font-family: Roboto;
+    src: url("../assets/fonts/Roboto-Light.ttf");
 }
+
 /* - - - - END - F O N T S - A N D - V A R S - END - - - - - */
 
 /* - - - - - G E N E R A L - - - - - */
 body{
 	margin: auto;
+	width: 95%;
 	max-width: 1400px;
```

Comment:
Is for visual, on screens less than 1400px, the width of the page is at 95% of the viewport. 
![Screenshot_1](https://user-images.githubusercontent.com/111531860/197206823-fc0ffdf3-6621-4298-b6a2-a87c1934d307.jpg)
Allows a minimun margin to the page, otherwise the images seems to be cut by the screen border